test/parallel/test-https-timeout.js

```diff
@@ -30,8 +30,8 @@ const https = require('https');
 const fs = require('fs');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${common.fixtures}/keys/agent1-key.pem`),
```

Comment:
`common` module does not export fixtures. This should be using method `fixtures.readKey` to get keys.

Reply:
Yeah, this is not quite correct. The change would be first to:

```js
const fixtures = require('../common/fixtures');
```

Then...

```js
const options = {
  key: fixtures.readKey('agent1-key.pem'),
  cert: fixtures.readKey('agent1-cert.pem')
}
```